ui/nuxeo-slots.js

```diff
@@ -180,12 +198,35 @@ window.nuxeo.slots.setSharedModel = (model) => {
       });
     }
 
+    /**
+     * Returns a pair with:
```

Comment:
nit: not a big fan of multiple return values... would rather have it return an object and people do destructuring assignment ie: `const { parent, parentSlot } = this._effectiveParent();`
still, assuming finding the slot after having the parent isn't tricky (or vice versa) would rather have two methods for this (or even just getters).

Reply:
The problem is that we may not have a `parentSlot`. If e do, it will be needed to compute the `parent`. If it doesn't, parent computation is straightforward. Computing both at the same time is more more efficient, so I don't know how introducing getters here would help. But will move it to an object instead an array. 👍